test/PublishCommand.js

```diff
@@ -166,6 +166,30 @@ describe("PublishCommand", () => {
       }));
   });
 
+  describe("normal mode - fixed version publish", () => {
+    let testDir;
+
+    beforeEach(async () => {
+      testDir = await initFixture("PublishCommand/normal");
+
+      GitUtilities.hasTags.mockReturnValue(true);
+      GitUtilities.getLastTag.mockReturnValue("v1.0.0");
+      GitUtilities.diffSinceIn.mockImplementation((since, location) => {
+        if (location.endsWith("package-3")) {
+          return "packages/package-3/newfile.json";
+        }
+        return "";
+      });
+    });
+    it("should publish all packages in Fixed Version mode", async () => {
+      await run(testDir)("--cd-version", "patch");
```

Comment:
This isn't at all what was discussed in the issue, patch or minor bumps should remain partial based on actual changes, as the local sibling dependencies are `^` ranges (by default).

Reply:
You are right. We haven’t reached 1.0 yet, so in our case, we need cater for minor bump as well

Reply:
Well, fun fact, below 1.0.0 the minor versions are treated as majors (e.g., `0.3.0` does not satisfy `^0.2.0`). So it's a bit more complicated than that.